js/dragquestion.js

```diff
@@ -868,6 +900,9 @@ H5P.DragQuestion = (function ($) {
     self.y = element.y;
     self.width = element.width;
     self.height = element.height;
+    if (self.height < 10) {
```

Comment:
After some reflection, I have determined that Cameron added this code to deal with a side effect of changing the units from em to %. I am going to remove it.